src/main/views/case-name-search.njk

```diff
@@ -0,0 +1,81 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "./macros/common-components.njk" import submitButton %}
+
+{% extends "template.njk" %}
+{% block pageTitle %}
+    {{ "Case Name Search" | safe }}
+{% endblock %}
+{% block beforeContent %}
+    {{ govukBackLink({
+        text: text,
+        href: '#',
+        attributes: {
+            onClick: "history.back();event.preventDefault();"
+        }
+    }) }}
+{% endblock %}
+{% block content %}
+    {% if noResultsError %}
+        {% set errorMessage = 'Case name not found. Please try again or search by case reference number, unique reference number, or by a whole court' %}
+        <div class="govuk-grid-row">
+            <div class="no_padding govuk-grid-column-full">
+                {{ govukErrorSummary({
+                    titleText: "There is a problem",
+                    errorList: [
+                        {
+                            text: errorMessage,
+                            href: "#case-name"
+                        }
+                    ]
+                }) }}
+            </div>
+        </div>
+    {% endif %}
+    {% set inputClasses = 'govuk-!-width-two-thirds govuk-input--error'
+        if noResultsError else 'govuk-!-width-two-thirds' %}
+    {% set formGroupError = 'govuk-form-group--error'
+        if noResultsError else 'govuk-form-group' %}
+    <form action="case-name-search" method="post">
+        <div class="govuk-form-group govuk-!-width-one-half">
+            <h1 class="govuk-heading-l">Enter a case name</h1>
+            {% if noResultsError %}
+                {{ govukInput({

```

Comment:
Can we have only one input component? Instead of having a 2 input in a if then else block we can set some variables for the error message and error classes and use them to build the input component. See search.njk as an example

Reply:
done